packages/ui/tailwind.config.js

```diff
@@ -5,5 +5,13 @@ module.exports = {
   ...sharedConfig,
   content : [
     "./src/**/*.{js,ts,jsx,tsx,mdx}"
-  ]
+  ],
+  theme:{
```

Comment:
@Gayathri-Nakkeeran , add these custom tailwind config to the tailwind config from packages/tailwind-config/tailwind.config.js file. So that the custom properties can be accessed in both the web app and ui packages.